docs/src/main/paradox/security/index.md

```diff
@@ -19,7 +19,6 @@ report with the Lightbend Akka team.
 
 ## Security Related Documentation
 
- * [Akka security fixes]($pekko.doc.dns$/docs/pekko/current/security/index.html)
```

Comment:
I think that this was meant to point to the Akka docs but was accidentally changed to be a Pekko link.

Could you keep this but change the link to https://akka.io/security/ ?

Reply:
I see.

Reply:
![image](https://github.com/apache/incubator-pekko/assets/501740/5b87e76d-d66e-492b-8bf9-968a21166618)

it's `

Reply:
updated, thank you